Intersect.Editor/Localization/Strings.cs

```diff
@@ -2042,6 +2047,10 @@ public struct EventConditional
 
             public static LocalizedString canstartquest = @"Can Start Quest";
 
+            public static LocalizedString equipmentslot = @"Slot";
```

Comment:
Change the name to `EquipmentSlot`, with that exact capitalization.

Doing all lowercase was a bad design choice by JC, when the strings can be referenced case-insensitively during loading/saving. We are trying to fix this with each new PR rather than repeat the mistake since it requires humans to fix the casing.

Reply:
I understand, I didn't know that, when you asked to rename it I thought it was just a writing habit, and I continued following the pattern I was, I will change